bindings/wasm/cypress/integration/revokeVC.js

```diff
@@ -0,0 +1,26 @@
+import {
+    defaultClientConfig,
+    initIdentity,
+    repeatAsyncTest,
+    revokeVC
+} from '../../examples/dist/web'
+
+describe(
+    "revokeVC",
+    {
+        defaultCommandTimeout: 1000*60*3, // 3 minutes to account for spurious network delays
+    },
+    () => {
+        before(async () => {
+            // The working directory is under __cypress at test runtime, so we need to go up one more level than usual
+            await initIdentity('../../../examples/dist/identity_wasm_bg.wasm');
+
+            // NOTE: `cy.wrap(defaultClientConfig()).as('config')` does not always work to make the config available
+            // from the shared context as `this.config` because it has a race condition with initializing the wasm.
+            // So call `defaultClientConfig()` manually for now.
+        });
+        it("Revoke Verifiable Credential", async () => {
+            await repeatAsyncTest(revokeVC, defaultClientConfig());
+        });
+    }
```

Comment:
Can we put some more of the boilerplate in a shared file to deduplicate it? Like the `defaultCommandTimeout` value and `initIdentity` stuff. Maybe also move `repeatAsyncTest` somewhere under the `cypress` directory too?

Reply:
It is being very finnicky with separating init and config. I suspect a problem with referential equality between Wasm and JS. I would suggest we revisit that with the account examples.